examples/cross.js

```diff
@@ -0,0 +1,69 @@
+var data = require('./data/data.json')
+var streamSift = require('../lib')
+var c = require('colors/safe')
+
+/**
+* $cross
+* $crossOrEqual
+*/
+
+var spec = { input : { $c : 1 } }
+var ss = streamSift(spec)
```

Comment:
Should be called `test` or something better. Maybe `isMatch`? Haven't found a great term yet.